object-construction-checker/tests/socket/OptionalSocket.java

```diff
@@ -0,0 +1,27 @@
+// A test case for some false positives that came up in Zookeeper.
+// These are sort-of a must call choice situation, but it's not as clear.
+// I suspect our MCC implementation won't actually handle these cases, in
+// which case it's fine with me to skip this test for now. - Martin
+
+import java.util.*;
+import java.net.*;
+import java.io.*;
+import org.checkerframework.checker.objectconstruction.qual.*;
+
+class OptionalSocket {
```

Comment:
Rather than adding these expected errors, we should skip this test and keep it correct. To do that, add the following line at the beginning of the file:

```
// @skip-test
```

I did the same with the `MustCallChoice` tests in the MustCallChecker's test directory that require us to actually implement checks that MCC is used correctly.